TourGuide_Horde/01_12_Durotar.lua

```diff
@@ -138,8 +138,8 @@ A Training the Beast |QID|6081| |C|Hunter|
 P Tame Venomtail Scorpid |C|Hunter|
 
 A Veteran Uzzek |QID|1505| |C|Warrior| |T|
-T Veteran Uzzek |QID|1505| |C|Warrior| |T|
-A Path of Defense |QID|1498| |C|Warrior| |T|
+T Veteran Uzzek |QID|1505| |C|Warrior|
+A Path of Defense |QID|1498| |C|Warrior|
```

Comment:
What was your reasoning for removing the |T| tags here? Uzzek is in Far Watch Post, right?

Reply:
I wasn't sure if the |T| tag needed to be in multiple steps if they were sequential. Will revert.